README.md

```diff
@@ -1,98 +1,119 @@
+to visualize the predictions and display them on screen with OpenCV:
 
-- **pip**: Installs `inference` into your Python environment. Lightweight, good for Python-centric projects.
-- **Docker**: Packages `inference` with its environment. Ensures consistency across setups; ideal for scalable deployments.
+```python
+import cv2
+import inference
+import supervision as sv
+
+annotator = sv.BoxAnnotator()
+
+inference.Stream(
+    source="webcam", # or rtsp stream or camera id
+    model="rock-paper-scissors-sxsw/11", # from Universe
+
+    output_channel_order="BGR",
+    use_main_thread=True, # for opencv display
+    
+    on_prediction=lambda predictions, image: (
+        print(predictions), # now hold up your hand: 🪨 📄 ✂️
+        
+        cv2.imshow(
+            "Prediction", 
+            annotator.annotate(
+                scene=image, 
+                detections=sv.Detections.from_roboflow(predictions)
+            )
+        ),
+        cv2.waitKey(1)
+    )
+)
 
-## 💻 install
+```
 
-### With ONNX CPU Runtime:
+## 👩‍🏫 More Examples
 
-For CPU powered inference:
+The [`/examples`](https://github.com/roboflow/inference/blob/main/examples/) directory contains code samples for working with and extending `inference` including using foundation models like CLIP, HTTP and UDP clients, and an insights dashboard, along with community examples (PRs welcome)!
 
-```bash
-pip install inference
-```
+## 🎥 Inference in action
 
-or
+Check out Inference running on a video of a football game:
 
-```bash
-pip install inference-cpu
-```
+https://github.com/roboflow/inference/assets/37276661/121ab5f4-5970-4e78-8052-4b40f2eec173
 
-### With ONNX GPU Runtime:
+## 💻 Why Inference?
 
-If you have an NVIDIA GPU, you can accelerate your inference with:
+Inference provides a scalable method through which you can manage inferences for your vision projects.
 
-```bash
-pip install inference-gpu
-```
+Inference is composed of:
 
-### Without ONNX Runtime:
+- Thousands of [pre-trained community models](https://universe.roboflow.com) that you can use as a starting point.
 
-Roboflow Inference uses Onnxruntime as its core inference engine. Onnxruntime provides an array of different [execution providers](https://onnxruntime.ai/docs/execution-providers/) that can optimize inference on differnt target devices. If you decide to install onnxruntime on your own, install inference with:
+- Foundation models like CLIP, SAM, and OCR.
 
-```bash
-pip install inference-core
-```
+- A tight integration with [Supervision](https://roboflow.com/supervision).
 
-Alternatively, you can take advantage of some advanced execution providers using one of our published docker images.
+- An HTTP server, so you don’t have to reimplement things like image processing and prediction visualization on every project and you can scale your GPU infrastructure independently of your application code, and access your model from whatever language your app is written in.
```

Comment:
`A HTTP` server.

Reply:
Must be British English :p “an” sounds right to me here.